recipes/opencv/4.x/conanfile.py

```diff
@@ -1515,6 +1513,11 @@ def generate(self):
                 tc.variables["WITH_OBSENSOR"] = False
             tc.variables["WITH_ZLIB_NG"] = False
 
+        if Version(self.version) >= "4.11.0":
+            tc.variables["WITH_HAL_RVV"] = False
+        else:
+            tc.variables["ENABLE_NEON"] = self.options.get_safe("neon", False)
```

Comment:
has something changed in 4.11.0 with regards of the handling of `ENABLE_NEON`? we can't understand the motivation behind this change

Reply:
As far as I understand, it's been removed in https://github.com/opencv/opencv/commit/4c81e174bfb28e67251d33ac812ba0cc57a2f3c4.